traffic_portal/app/src/common/modules/form/origin/form.origin.tpl.html

```diff
@@ -0,0 +1,147 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="x_panel">
+    <div class="x_title">
+        <ol class="breadcrumb pull-left">
+            <li><a ng-click="navigateToPath('/origins')">Origins</a></li>
+            <li class="active">{{originName}}</li>
+        </ol>
+        <div class="clearfix"></div>
+    </div>
+    <div class="x_content">
+        <br>
+        <form name="originForm" class="form-horizontal form-label-left" novalidate>
+            <div ng-show='!settings.isNew' class="form-group">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Primary:</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input name="isPrimary" type="text" class="form-control" ng-value="origin.isPrimary" readonly autofocus>
```

Comment:
is this a true|false? do you want to use a select instead?

Reply:
`isPrimary` is a read-only field, so I just wanted TP to display it here without actually sending the field back in the PUT/POST request.